apps/console/src/features/authentication/hooks/use-sign-in.ts

```diff
@@ -278,42 +278,49 @@ const useSignIn = (): UseSignInInterface => {
             }
         }
 
-        // FIXME: Skipping /o/ appending from the `getServiceResourceEndpoints` level seems to be not working.
-        const wellKnownEndpoint: string = Config.getServiceResourceEndpoints().wellKnown.replace("/o/", "/");
-
-        // Set configurations related to hostname branding.
-        axios
-            .get(wellKnownEndpoint)
-            .then((response: AxiosResponse) => {
-                // Use token endpoint to extract the host url.
-                const splitted: string[] = response?.data?.token_endpoint?.split("/") ?? [];
-
-                let serverHost: string = splitted.slice(0, -2).join("/");
-
-                if (orgType === OrganizationType.SUBORGANIZATION) {
-                    serverHost = `${Config.getDeploymentConfig().serverOrigin}/${
-                        window["AppUtils"].getConfig().organizationPrefix
-                    }/${window["AppUtils"].getConfig().organizationName}`;
-                }
+        let wellKnownEndpoint: string = Config.getServiceResourceEndpoints().wellKnown;
+        const disabledFeatures: string[] = window["AppUtils"].getConfig()?.ui?.features?.branding;
```

Comment:
Lets use the redux selector.

```ts
const disabledFeatures: string[] = useSelector(...
```

Reply:
Fixed with 38f1ee2029dbf863e1f5ad8490a2bab8c4985ebe.